### ISSUE TITLE
Timber::get_posts doesn't support `category`

### ISSUE THREAD

**Issue by connorjburton** [timber/timber#951]
### Expected behavior

Okay, this is a weird one and I just spent way too long trying to figure it out. The issue comes from Wordpress, not Timber but I think it's something we can solve. Very simple:

`get_posts` accepts `category => id` as an argument. It does not support `cat => id` as an argument

`WP_Query` (which we use internally for `Timber::get_posts`) accepts `cat => id` as an argument. It does not support `category => id` as an argument.

So if you are like me and just assumed the same parameters I used to use in `get_posts` will work in `Timber::get_posts` this can cause a very confusing issue.

### Steps to reproduce behavior

1. Create a post
2. Assign a category
3. In a PHP file, `var_export(get_posts(array('category' => IDOFCAT)));`
4. In the same PHP file, `var_export(Timber::get_posts(array('category' => IDOFCAT)));`

The second `var_export` won't return what is expected. As you can see, very confusing, same method name, not the same result.

### What version of WordPress, PHP and Timber are you using?

Latest all

### How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)

Composer

**Comment by johnstew** [comment]
Yea so I can confirm this behavior. I was running into the same issue for the past couple of days and had no clue until I read this that the arguments differ. I thought they were the exact same as WP Query. 

I'm not sure if I missed this in the documentation but I think it would be best moving forward to have the same argument expectations for `Timber::get_posts` as we do for `WPQuery`.

**Comment by johnstew** [comment]
Okay so after more research here I realized that there actually is an example of how to do a query on with Taxonomy right in from of my face the entire time.

Look at the query:
https://github.com/timber/timber/wiki/Timber-docs#get_posts

However, it still is a little misleading because it says that it supports a WP_Query args but from my attempts at using the [category parameters](https://codex.wordpress.org/Class_Reference/WP_Query#Category_Parameters). It looks like there is some sort of disconnect on what is accepted for these. 

Long story short, use the query in the docs under `get_posts` and also reference [WP Tax Parameters](https://codex.wordpress.org/Class_Reference/WP_Query#Taxonomy_Parameters) for any further needs. This approach got me the correct data returned after days of trying other ways.

**Comment by connorjburton** [comment]
Under the hood `Timber::get_posts` uses `WP_Query`, however that accepts different parameters than `get_posts`, which causes the confusion.

**Comment by jarednova** [comment]
close by #1157
